protocols/s7/src/main/resources/protocols/s7/s7.mspec

```diff
@@ -897,6 +897,7 @@
     ['0x0B' UDINT          ['0x07'     , 'D'             , '4'               , 'INT'                 , 'INTEGER'                          , 'IEC61131_UDINT'          , 'false'             , 'false'             , 'true'               , 'true'               , 'true'              ]]
     ['0x0C' LINT           ['0x00'     , 'X'             , '8'               , 'INT'                 , 'BYTE_WORD_DWORD'                  , 'IEC61131_LINT'           , 'false'             , 'false'             , 'false'              , 'true'               , 'false'             ]]
     ['0x0D' ULINT          ['0x00'     , 'X'             , '8'               , 'INT'                 , 'BYTE_WORD_DWORD'                  , 'IEC61131_ULINT'          , 'false'             , 'false'             , 'false'              , 'true'               , 'false'             ]]
+    ['0x1C' COUNTER        ['0x1C'     , 'X'             , '2'               , 'null'                , 'OCTET_STRING'                  , 'IEC61131_INT'           , 'true'              , 'true'              , 'true'               , 'true'               , 'false'             ]]
```

Comment:
Hmmm it seems COUNTER has the same code as DATE_AND_TIME ... does that mean that support of COUNTER now breaks the ability to read DATE_AND_TIME?

Reply:
Hi Chris,

Yes, but the enum type generated by the mspec does the magic.

Attached screenshot for a DATE_AND_TIME,

Best regards,



![imagen](https://github.com/user-attachments/assets/1c199008-8b14-469a-a507-604943bc1b46)

![imagen](https://github.com/user-attachments/assets/4a9bee6b-da97-432e-aa37-dcdf6768eb6a)

Reply:
Ok ... so I tested my Manual tests with the devices that I have and I didn't run into any problems. So please ignore this comment ;-)